include/openthread/thread_ftd.h

```diff
@@ -739,6 +739,19 @@ void otThreadRegisterNeighborTableCallback(otInstance *aInstance, otNeighborTabl
  */
 void otThreadSetCcmEnabled(otInstance *aInstance, bool aEnabled);
 
+/**
+ * This method sets flag informing that thread protocol version check VR should be skipped
+ * when deciding if device should operate as router.
+ *
+ * @note This API requires `OPENTHREAD_CONFIG_REFERENCE_DEVICE_ENABLE`, and is only used by Thread Test Harness
+ *       to indicate that thread protocol version check VR should be skipped.
+ *
+ * @param[in]  aInstance  A pointer to an OpenThread instance.
+ * @param[in]  aEnabled  TRUE if the VR check should be skipped, FALSE otherwise.
+ *
+ */
```

Comment:
```suggestion
/**
 * This function sets whether the Security Policy TLV version-threshold for routing (VR field) is enabled.
 *
 * @note This API requires `OPENTHREAD_CONFIG_REFERENCE_DEVICE_ENABLE`, and is only used by Thread Test Harness
 *       to indicate that thread protocol version check VR should be skipped.
 *
 * @param[in]  aInstance  A pointer to an OpenThread instance.
 * @param[in]  aEnabled   TRUE to enable Security Policy TLV version-threshold for routing, FALSE otherwise.
 *
 */
```